docs/source/routing/self-hosted/containerization/aws.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Deploying GraphOS Router on Amazon Web Services
+subtitle: Deploy router with Amazon Elastic Container Service (ECS) 
+description: Build and deploy Apollo Router on Amazon Web Services (AWS) with Amazon Elastic Container Service (ECS)
+---
+
+Learn how to deploy the router for development on AWS with Elastic Container Service (ECS).
+
+You will:
+- Build a router image with a Dockerfile.
+- Set up an Elastic Cloud Registry and push your router image to it.
+- Create an ECS task definition for your router and deploy it.
+
+<Note>
+
+Not deploying to AWS? Try our router deployment guides for [GCP](/graphos/routing/self-hosted/containerization/gcp) or [Azure](/graphos/routing/self-hosted/containerization/azure).
+
+</Note>
+
+## Prerequisites
+
+Before you start:
+
+1. [Set up a GraphQL API in GraphOS](/graphos/get-started/guides/graphql#step-1-set-up-your-graphql-api).
+    - Save your `APOLLO_KEY` and `APOLLO_GRAPH_REF`. You'll need them when deploying the router.
+1. Install [Docker](https://www.docker.com/get-started/) locally.
+1. Set up you [AWS environment](https://aws.amazon.com/getting-started/guides/setup-environment/)
+    - Install the AWS CLI.
+    - Use an existing Create an Amazon 
+1. Choose a version of the router to deploy (for example, `v1.61.0`). You'll need it when specifying the router image to deploy.
+
+## Build router image
+
+To deploy your own router, start by customizing and building a router image, using a Dockerfile and a router configuration file:
+
+1. In a local directory, create a `router.yaml` file and copy-paste the following configuration into the file: 
+    
+    ```yaml title="router.yaml"
+    supergraph:
+        listen: 0.0.0.0:4000
+    health_check:
+        listen: 0.0.0.0:8088
+    ```
+
+    The router's default HTTP and health check endpoint addresses are localhost, so they wouldn't be reachable when deployed. This configuration enables the router to listen to all addresses.
+
+1. Create a `Dockerfile` file and copy-paste the following into the file:
```

Comment:
Until this point I've seen "Dockerfile" w/o backticks. Is there a reason to put it in `<code>` here?